orienteer-core/src/main/java/org/orienteer/core/component/command/DeleteODocumentCommand.java

```diff
@@ -49,10 +36,11 @@ public DeleteODocumentCommand(OrienteerDataTable<ODocument, ?> table, IModel<OCl
 	@Override
 	protected void performMultiAction(AjaxRequestTarget target, List<ODocument> objects) {
 		super.performMultiAction(target, objects);
-		ODatabaseDocument db = getDatabase();
-		db.commit(true);
-		db.begin();
-		db.getMetadata().reload();
+		DBClosure.sudoConsumer(db -> {
+			db.commit(true);
```

Comment:
Why is it here? We need to commit in initial db, but not in admin's one.

Reply:
You are right. I fixed it in dc304730cd6af51d30887555ba62c0c1c4cfd70a